near-sdk/src/store/lookup_set/mod.rs

```diff
@@ -1,54 +1,64 @@
 mod impls;
 
 use crate::store::key::{Identity, ToKey};
-use crate::{env, IntoStorageKey, StableMap};
+use crate::{env, IntoStorageKey};
 use borsh::{BorshDeserialize, BorshSerialize};
-use once_cell::unsync::OnceCell;
 use std::borrow::Borrow;
 use std::fmt;
 use std::marker::PhantomData;
 
+/// A non-iterable implementation of a set that stores its content directly on the storage trie.
+///
+/// This set stores the values under a hash of the set's `prefix` and [`BorshSerialize`] of the
+/// value and transformed using the map's [`ToKey`] implementation.
```

Comment:
I think you meant to say "key" here

```suggestion
/// This set stores the values under a hash of the set's `prefix` and [`BorshSerialize`] of the
/// key and transformed using the map's [`ToKey`] implementation.
```

Reply:
Oh, this is a set, ignore me :see_no_evil: